ui/src/screens/About/FAQScreen.tsx

```diff
@@ -57,17 +57,83 @@ const CaretSVG = styled.svg<{ collapsed: boolean }>`
     answer:
-      "Click the Connect with button on the talents board to launch a message screen. Then, send some useful information \
-    about the job like a Job Description or a Company Website. The talent will receive your message via email.",
+      "'Sign In' using your LinkedIn account and create a Jobseeker profile. Click on 'I'm Seeking' on Profile Type and \
+      fill the required details to complete your profile. You are required to add a link to your portfolio/resume/ GitHub \
+      profile URL so that potential recruiters can learn more about you.",
+    collapsed: true,
+  },
+  {
+    question: "How do I sign in as a recruiter?",
+    answer:
+      "'Sign In' using your LinkedIn account and create a Recruiter profile. Click on 'I'm Hiring' \
+      on Profile Type and fill the required details to complete your profile. Add your Job Title and \
+      Company Name so that potential job seekers can learn more about you.",
+    collapsed: true,
+  },
+  {
+    question: "I don't have a LinkedIn account. Is there any other way I can sign in?",
+    answer:
+      "At this moment, we are using only LinkedIn to authenticate users. In future, \
+      we may add other Sign-In options.",
+    collapsed: true,
+  },
+  {
+    question: "Can I be both job seeker and recruiter?",
+    answer:
+      "As the platform aims to establish a connection between job seekers and recruiters, \
+      we are only allowing an individual to be a seeker or recruiter, not both.",
+    collapsed: true,
+  },
+  {
+    question: "Do I need to pay to use your service?",
+    answer:
+      "The service is FREE for both recruiters and jobseekers. We aim to keep it this way forever. We don't intend \
+      to make any revenue out of this platform.",
+    collapsed: true,
+  },
+  {
+    question: "Why am I unable to update my email address?",
+    answer:
+      "As we use LinkedIn as our authentication service, we are taking the email that is \
+      provided by LinkedIn. This ensures that the email is verified and the person has a \
+      LinkedIn profile guaranteeing the authenticity of an individual. We are not allowing \
+      users to change their emails at the moment. ",
+    collapsed: true,
+  },
+  {
+    question: "What is the site redirecting me to Auth0?",
+    answer:
+      "Auth0 is an industry-leading solution that provides authentication and authorization \
+      services. By using Auth0, we have reduced our development effort at least by a month or \
+      two. In addition to reducing the development time, Auth0 also provides us with additional \
+      security features and protect us by blocking suspicious IP addresses and avoids DDoS attacks. \
+      In short, by using Auth0, your account is more secure. ",
+    collapsed: true,
+  },
+  {
+    question: "How do I connect with a Job provider?",
+    answer:
+      "Click the Connect with button on the jobs board to launch a message screen. Type in your \
```

Comment:
@subhransu Did you mean the to wrap `Connect with button` in quotes, like `'Connect with...' button`.
Looks abit awkward now I think.